### PR TITLE
fix(ipns): ensures the latest path is stored on a triggered publish

### PR DESCRIPTION
**Pull request by andrewxhill**
Signed-off-by: Andrew Hill <andrew@textile.io>

### CODE REVIEW

**Review by carsonfarmer — APPROVED** [review]
Good catch

File: api/apitest/apitest.go [hunk 1]
```diff
@@ -42,22 +42,23 @@ func DefaultTextileConfig(t util.TestingTWithCleanup) core.Config {
...
+		AddrAPI:                    util.MustParseAddr(fmt.Sprintf("/ip4/127.0.0.1/tcp/%d", apiPort)),
+		AddrAPIProxy:               util.MustParseAddr("/ip4/0.0.0.0/tcp/0"),
+		AddrMongoURI:               GetMongoUri(),
+		AddrMongoName:              util.MakeToken(12),
+		AddrThreadsHost:            util.MustParseAddr("/ip4/0.0.0.0/tcp/0"),
+		AddrIPFSAPI:                GetIPFSApiAddr(),
+		AddrGatewayHost:            util.MustParseAddr(fmt.Sprintf("/ip4/127.0.0.1/tcp/%d", gatewayPort)),
+		AddrGatewayURL:             fmt.Sprintf("http://127.0.0.1:%d", gatewayPort),
+		IPNSRepublishSchedule:      "0 1 * * *",
+		MaxRepublishingConcurrency: 5,
```
Comment (sanderpick) [inline api/apitest/apitest.go:56]: Can we make this match the other related param, like `IPNSRepublishConcurrency`?

File: cmd/buckd/main.go [hunk 2]
```diff
@@ -86,6 +86,10 @@ var (
 				Key:      "ipns.republish_schedule",
 				DefValue: "0 1 * * *",
 			},
+			"maxRepublishingConcurrency": {
+				Key:      "ipns.max_republishing_concurrency",
```
Comment (sanderpick) [inline cmd/buckd/main.go:90]: and here, `ipns.republish_concurrency`?

**Review by sanderpick — APPROVED** [review]

File: api/apitest/apitest.go (hunk 1 above)
Comment (andrewxhill) [reply api/apitest/apitest.go]: done

File: cmd/buckd/main.go (hunk 2 above)
Comment (andrewxhill) [reply cmd/buckd/main.go]: and done

**Pull request merged**